### ISSUE TITLE
Trying to use "fireducks.pandas" explicit import in Linux Mint, but keep getting error message. 

### ISSUE THREAD

**Issue by RgC2023** [fireducks-dev/fireducks#23]
Hi, 

I have recently installed Linux Mint 22 onto a computer, with Python 3.12 pre-installed. I had to create a virtual environment in Python in order to use the "pip install fireducks" command in Linux Mint 22 successfully. However, when I create a Python file in the virtual environment folder in Python, and use the "import fireducks.pandas as pd" command in the Python file to create a pandas dataframe faster, I get an error message saying : "ModuleNotFoundError: No module named 'fireducks'". 

I checked to see if "fireducks" was installed in the virtual environment folder with pip, and it was installed. I tried using "import fireducks as pd" - still doesn't work. The same error occurs when I try to run the python file  in either my home folder or the virtual environment folder.

Can someone explain to me what's going on, and how to fix this issue? I would greatly appreciate it if someone helps me resolve this issue as soon as possible.

**Comment by bryanpiguave** [comment]
Hi! 
I had a similar problem with another package. Here is what I did to debug it.:
First, activate your virtual environment:  source /path/to/your/virtualenv/bin/activate
Then run which python to see the path of the Python interpreter. 
Additionally, check that you are using the same Python in which Fireducks was installed. 
I hope this could help you.

**Comment by RgC2023** [comment]
Hi, 

I have checked the path of the python interpreter in my virtual environment. The path in my virtual environment is ".../bin/python". The path of the python interpreter I use outside the virtual environment is ".../bin/python3". I noticed that in the pathway to my python interpreter in my virtual environment, there is "python", "python3", and "python3.12"; the pathway to my python interpreter outside my virtual environment leads me to "python3" and "python3.12". 

Given all of this information, what am I supposed to do now? Am I supposed to somehow switch between "python" and "python3" in my virtual environment to match what I have outside my virtual environment? If so, how do I do that? 

Also, how do I check that I am using the same Python in which Fireducks has been installed? Is it a command in the Linux Mint 22 command terminal?

To all of these questions, I would greatly appreciate your help in resolving them. Thank you for your help so far!

**Comment by RgC2023** [comment]
Hi there!

I believe I have solved my problem! 

The "Fireducks" module needed to be installed with "pip", and thus needed to be installed inside a python virtual environment on Linux Mint 22. This is because Linux Mint 22 has seemingly disallowed python module "pip" installations outside of a virtual environment, in favor of python module installations using "debian" and "apt install python3-modulename" (if said module is available through "debian") in the command line. Thus, the only way that Fireducks could run on Linux Mint 22 command line is if I was inside my virtual environment, not outside of it. The same is true with any Python package/module that needed to be installed with "pip". All those that can only be installed with "debian" on Linux Mint 22, and cannot be installed through "pip", can only be run outside the virtual environment.

I found the correct pathway where "fireducks" was installed - the pathways I mentioned in my last comment were the wrong ones. 

Thank you for your help, and for aiding me in finding the solution to this problem!
